src/documents/tutorial-prefsFramework/CreatingAPrefsEditor.md

```diff
@@ -0,0 +1,779 @@
+provisions for an iframe in the page to preview any changes made by the Editor.
+
+In the code snippet above, the `loaderGrades` option is used to specify the “full page, no preview” form.
+
+##### Templates #####
+The Auxiliary Schema must declare where to find the main HTML template for the Preference Editor.
+In our example, this template is located in the same folder as other HTML templates.
+The Auxiliary Schema allows you to define `terms` – strings that can be re-used elsewhere in the
+schema. Here, it is being used to define, in a single place, the path to where the HTML templates are:
+
+```javascript
+    terms: {
+        templatePrefix: "html"
+    },
+```
+
+The template property specifies the main HTML template for the entire Preference Editor:
+
+```javascript
+    template: "%templatePrefix/minEditor.html",
+```
+You can see the full text of this file, `minEditor.html`, in the github repo:
+https://github.com/fluid-project/infusion/tree/master/examples/framework/preferences/minimalEditor/html/minEditor.html
+The main thing to note in the template is the placeholder for the Panel,
+in this example a `<div>` with the class `mec-autoPilot”`:
+
+```html
+<!-- placeholder for the auto-pilot preference panel -->
+<div class="mec-autoPilot"></div>
+```
+
+The Framework will insert the constructed Panel into this div.
+
+##### Preferences #####
+The next thing in the Auxiliary Schema is the configuration for the auto-pilot preference:
+
+```javascript
+autoPilot: {
+    // this 'type' must match the name of the pref in the primary schema
+    type: "minEditor.autoPilot",
+
+    panel: {
+        // this 'type' must match the name of the panel grade created for this pref
+        type: "minEditor.panels.autoPilot",
+
+        // selector indicating where, in the main template, to place this panel
+        container: ".mec-autoPilot",
+
+        // the template for this panel
+        template: "%templatePrefix/autoPilot.html"
+    }
+}
+```
+
+(The name of the property, `autoPilot`, can actually be anything, but it’s helpful to use the name
+of the preference.)
+
+In our example, the auto-pilot preference configuration includes two things:
+1. the type of the preference, and
+2. information about the Panel.
+
+The value of the `type` property is the name of the preference as defined in the Primary Schema.
+
+The value of the `panel` property is a JavaScript object containing configuration information
+for the Panel. Let’s look at each of the properties:
+<dl>
+<dt>`type`</dt>
+<dd>This is the name of the Panel that was defined in the call to `fluid.defaults()` above.</dd>
+<dt>`container`</dt>
+<dd>This is a CSS-based selector referencing the Panel’s placeholder element in the main HTML
+template – the one referenced by the `template` property above.</dd>
+<dt>`template`</dt>
+<dd>This is the path and filename of the HTML template for this Panel.
+Note, in this example, how the `templatePrefix` term is being used.</dd>
```

Comment:
Should this be a note styled as specified in [the style guide](https://wiki.fluidproject.org/display/fluid/Infusion+Documentation+Style+Guide)? If not, maybe change "Note" to "Notice"